### ISSUE TITLE
v0.11.x behavior

### ISSUE THREAD

**Issue by lburgazzoli** [OrbitzWorldwide/consul-client#120]
I've found that the new v0.11.x behave differently from previous versions i.e.:
- for key/value keys starting with "/" lead to http 404
- for events the payload is surrounded by double quotes i.e one of my test cases failed becasue:
```
Expected :30b2f445-0000-4256-8aae-bc1578737972
Actual   :"30b2f445-0000-4256-8aae-bc1578737972"
```

Is that expected ?

**Comment by rickfast** [comment]
No, but it's a completely different http client so it's not unexpected. I will fix this morning

**Comment by rickfast** [comment]
Unexpected = surprising

**Comment by rickfast** [comment]
fixed by b64f7c6c1035bf1f5a60a1a7939ca60aca37fe25
